src/Mvc/test/WebSites/BasicWebSite/RequestIdMiddleware.cs

```diff
@@ -26,7 +27,7 @@ public Task Invoke(HttpContext context)
                 throw new InvalidOperationException("RequestId should be null here");
             }
 
-            var requestId = context.Request.Headers["RequestId"];
```

Comment:
Note: this was `"RequestId"` but `HostingApplicationDiagnostics.cs` uses `"Request-Id"`; now they are both using the same value via `HeaderNames.RequestId` (of `"Request-Id"`)